### PR TITLE
Bump runtime to 21.08

### PR DESCRIPTION
**Pull request by bbhtt**
- Add patch to fix compilation under GCC11. Patch is from
  https://src.fedoraproject.org/rpms/Ri-li/blob/rawhide/f/Ri-li-gcc11.patch

### CODE REVIEW

File: net.sourceforge.Ri-li.json [hunk 1]
```diff
@@ -29,13 +29,17 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://downloads.sourceforge.net/project/ri-li/Ri-li%20Linux_Unix/Ri-li%20V2.0.1/Ri-li-2.0.1.tar.bz2",
+                    "url": "http://downloads.sourceforge.net/sourceforge/ri-li/Ri-li-2.0.1.tar.bz2",
```
Comment (hfiguiere) [inline net.sourceforge.Ri-li.json]: this should be available in https too. It's prefered.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: net.sourceforge.Ri-li.json (hunk 1 above)
Comment (bbhtt) [reply net.sourceforge.Ri-li.json]: Oh yea, thanks.

**Comment by bbhtt** [comment]
bot, build

**Comment by bbhtt** [comment]
128px icon from Ubuntu package ri-li-data; 64px is down-scaled from that.

**Review by hfiguiere — APPROVED** [review]

**Pull request merged**